Automattic/jetpack
# Blogging prompts: add setting to enable or disable prompts in the editor

#### Changes proposed in this Pull Request:

Adds a setting to Settings > Writing to enable or disable writing prompts in the editor.

Companion PR in Calypso: https://github.com/Automattic/wp-calypso/pull/70739

#### Other information:


#### Jetpack product discussion

See discussions here: p7DVsv-fOI-p2

#### Does this pull request change what data or activity we track or use?

No.


#### Testing instructions:

- See new setting checkbox on Settings > Writing in wp-admin.
- By default, the box will be checked when the site has the `site_intent` option set to `write`, is set to show posts on the front page, or has a dedicated posts page set in Settings > Reading.
- By checking or unchecking the setting, it overrides the default.
- When enabled, a daily writing prompt will show as a placeholder when starting a new post.

<img width="1274" alt="Screen Shot 2022-12-05 at 10 09 12" src="https://user-images.githubusercontent.com/1699996/205685521-9c0d896d-4502-4098-a6ea-d4f2e7146be9.png">

- To test the API changes, follow the testing instructions in https://github.com/Automattic/wp-calypso/pull/70739
